test/test_ev.py

```diff
@@ -67,6 +67,7 @@ def test_validate_redraw_screen_enqueue(self):
             ui=self.mock_ui, config=self.conf,
             loop=self.mock_loop)
         dc.initialize = MagicMock()
+        dc.juju_state = MagicMock()
```

Comment:
Without it test_ev would fail:

```
======================================================================
ERROR: Validate redraw_screen on enqueue_deployed_charms
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/adam/Projects/openstack/test/test_ev.py", line 71, in test_validate_redraw_screen_enqueue
    dc.enqueue_deployed_charms()
  File "/home/adam/Projects/openstack/cloudinstall/core.py", line 563, in enqueue_deployed_charms
    self.wait_for_deployed_services_ready()
  File "/home/adam/Projects/openstack/cloudinstall/core.py", line 543, in wait_for_deployed_services_ready
    while not self.juju_state.all_agents_started():
nose.proxy.AttributeError: 'NoneType' object has no attribute 'all_agents_started'
```